### ISSUE TITLE
Yahoo Invalid Cookie

### ISSUE THREAD

**Issue by joelishness** [ghostfolio/ghostfolio#2006]
UI shows an ! with the following popup upon click:
Data Provider Errors for
ETHUSD (YAHOO)

Logs
```
docker logs ghostfolio-development_ghostfolio_1
[Nest] 306  - 05/27/2023, 1:48:00 AM   ERROR [YahooFinanceService] Error: Invalid Cookie
[Nest] 306  - 05/27/2023, 1:48:01 AM    WARN [PortfolioCalculator] Missing initial value for symbol ETHUSD at 2022-06-18
[Nest] 306  - 05/27/2023, 1:48:01 AM   ERROR [YahooFinanceService] Error: Invalid Cookie
```

I have seen other projects having issues with Yahoo API, also

**Comment by dtslvr** [comment]
Hello @joelishness 

This issue has been fixed in Ghostfolio [1.268.0](https://github.com/ghostfolio/ghostfolio/releases/tag/1.268.0). You probably just need to upgrade your instance.

Ps. Please use the bug report template next time. Thank you.

**Comment by dtslvr** [comment]
Duplicate of #1873.
